libtiledbsc/cmake/Modules/FindSpdlog_EP.cmake

```diff
@@ -130,7 +130,8 @@ elseif(TARGET spdlog::spdlog)
 endif()
 
 # If we built a static EP, install it if required.
-if (TILEDB_SPDLOG_EP_BUILT AND TARGET spdlog::spdlog)
-  include(TileDBCommon)
-  install_target_libs(spdlog::spdlog)
-endif()
+# TODO: Fix this block if needed, it breaks CMAKE_BUILD_TYPE=Coverage
+#if (TILEDB_SPDLOG_EP_BUILT AND TARGET spdlog::spdlog)
```

Comment:
Do we want this commented out?

Reply:
Ideally, we don't want this commented out. It causes a failure when building a `Coverage` build though. I can look into fixing it now.

Reply:
This block of cmake code came from the TileDB core library. In the core, it's used to conditionally install a static version of spdlog when building a static version of tiledb. Since we don't have the option of building a static version of tiledbsc, I removed this block of code.